ipn/ipnlocal/network-lock.go

```diff
@@ -474,6 +489,39 @@ func (b *LocalBackend) NetworkLockModify(addKeys, removeKeys []tka.Key) (err err
 	return nil
 }
 
+// NetworkLockDisable disables network-lock using the provided disablement secret.
+func (b *LocalBackend) NetworkLockDisable(secret []byte) error {
+	if err := b.CanSupportNetworkLock(); err != nil {
+		return err
+	}
+
+	var (
+		ourNodeKey key.NodePublic
+		head       tka.AUMHash
+	)
+
+	b.mu.Lock()
```

Comment:
optional: Move this into a separate function so you can defer the unlock? Alternatively, perhaps set an `err` to the error and return if non-nil after the unlock.

Reply:
Done the latter.